packages/server/src/notification/notification.service.ts

```diff
@@ -11,65 +11,38 @@ export class NotificationService {
     this.notificationUrl = this.configService.getOrThrow('NOTIFICATION_SERVICE_URL');
   }
 
-  async sendPasswordResetEmail(projectId: string, email: string, link: string): Promise<any> {
+  private async sendEmail(email: string, subject: string, message: string, template: string, projectId?: string, link?: string): Promise<any> {
```

Comment:
Although this is hard coded we should make this function more flexible.
- email: string, subject: string, message: string, template: string, are all part of an email structure. So you could put this into a single variable and type it above. As wells as other fields not being used like cc or bcc.
- to: [email], using a single email blocks the use of sending multiple people emails. 
- Auth send link mainly but template data could be anything and is unique to the template itself should be on the generic function you shouldn't pass in named template vars.
```suggestion
  private async sendEmail(email: Email, projectId?: string, templateData: any): Promise<any> {
```